### PR TITLE
ADEN-5510 Add biders to BLB

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/ADEN-5510

### CODE REVIEW

File: extensions/wikia/AdEngine/js/context/slotsContext.js [hunk 1]
```diff
@@ -28,23 +28,28 @@ define('ext.wikia.adEngine.context.slotsContext', [
...
 
-		setStatus('PREFOOTER_MIDDLE_BOXAD', isHome);
-
 		// those slots exists on all pages
 		setStatus('TOP_LEADERBOARD', true);
 		setStatus('TOP_RIGHT_BOXAD', true);
 
-		setStatus('LEFT_SKYSCRAPER_2', !isHome);
-		setStatus('LEFT_SKYSCRAPER_3', !isHome);
+		setStatus('PREFOOTER_MIDDLE_BOXAD', !isPremiumAdLayoutEnabled && isHome);
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/context/slotsContext.js]: PremiumAdLayout does not work on home pages anyway. `isHome` is sufficient

Comment (dianafa) [reply extensions/wikia/AdEngine/js/context/slotsContext.js]: 👍

File: extensions/wikia/AdEngine/js/context/slotsContext.js [hunk 2]
```diff
@@ -28,23 +28,28 @@ define('ext.wikia.adEngine.context.slotsContext', [
...
 		setStatus('INCONTENT_BOXAD_1', !isHome);
 
-		setStatus('INVISIBLE_HIGH_IMPACT_2', geo.isProperGeo(instantGlobals.wgAdDriverHighImpact2SlotCountries));
-		setStatus('PREFOOTER_RIGHT_BOXAD', !context.opts.overridePrefootersSizes);
+		setStatus('INVISIBLE_HIGH_IMPACT_2', !isPremiumAdLayoutEnabled && geo.isProperGeo(instantGlobals.wgAdDriverHighImpact2SlotCountries));
+		setStatus('PREFOOTER_RIGHT_BOXAD', !isPremiumAdLayoutEnabled && !context.opts.overridePrefootersSizes);
+		setStatus('PREFOOTER_LEFT_BOXAD', !isPremiumAdLayoutEnabled);
 
 		setStatus('INCONTENT_PLAYER', isIncontentEnabled);
+		setStatus('BOTTOM_LEADERBOARD', isPremiumAdLayoutEnabled);
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/context/slotsContext.js]: This one line is confusing me 🙈 . BLB is applicable in UAP scenario (no matter if it's premium ad layout)

**Comment by fraszczakszymon** [comment]
Should we add 970x250 size too?

**Comment by fraszczakszymon** [comment]
LGTM

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/indexExchange.js [hunk 3]
```diff
@@ -72,7 +72,16 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.indexExchange',[
 					],
 					id: '9',
 					siteID: 185049
-				}
+				},
+				// TODO: fill with proper numbers https://wikia-inc.atlassian.net/browse/ADEN-5517
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/indexExchange.js:76]: INFO Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/aol.js [hunk 4]
```diff
@@ -59,6 +59,14 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.aol', [
 					alias: '4431475',
 					sizeId: '529'
 				}
+				// TODO: fill with proper numbers https://wikia-inc.atlassian.net/browse/ADEN-5517
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/aol.js:62]: INFO Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/openx.js [hunk 5]
```diff
@@ -66,6 +66,14 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.openx',[
 					],
 					unit: 538735694
 				}
+				// TODO: fill with proper numbers https://wikia-inc.atlassian.net/browse/ADEN-5517
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/openx.js:69]: INFO Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![INFO][INFO] 3 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

**Pull request merged**